pages/api/route.js

```diff
@@ -0,0 +1,56 @@
+import { NextResponse } from "next/server";
+
+const API_KEY = process.env.DATA_API_KEY;
+
+export async function GET(URL) {
+    const res = await fetch(URL);
+    const data = await res.json();
+    return NextResponse.json(data);
+}
+export async function DELETE(URL, id) {
+    if (!id) return NextResponse.json({ "message": "No ID provided" });
+    await fetch(`${URL}/${id}`, {
+        method: 'DELETE',
+        headers: {
+            'Content-Type': 'application/json',
+            'API-KEY': API_KEY,

```

Comment:
remove api_key

Reply:
replace it with authorisation : Bearer token_xxxxx will be saved in the local storage

Reply:
> replace it with authorisation : Bearer token_xxxxx will be saved in the local storage

how we get this token ?